airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -495,6 +495,78 @@ def create_external_table(self,
                 'BigQuery job failed. Error was: {}'.format(err.content)
             )
 
+    def patch_table(self,
+                    project_id,
+                    dataset_id,
+                    table_id,
+                    schema=None,
+                    view=None):
+        """
+        Patch information in an existing table/view.
+        Schema changes can only be applied to tables, not views.
```

Comment:
This needs to support more than just updating schema as done by the api https://cloud.google.com/bigquery/docs/reference/rest/v2/tables/patch.